std/traits.d

```diff
@@ -388,7 +388,44 @@ version (none) version(unittest) //Please uncomment me when changing packageName
 }
 
 /**
- * Get the module name (including package) for the given symbol.
+ * Get the module name (including package) for the given symbol,
+ * returning it as a string.
+ *
+ * Example:
+ *    This can be used to implement something akin to
+ *    $(LINK2 https://en.wikipedia.org/wiki/Argument-dependent_name_lookup, Argument Dependent Lookup):
```

Comment:
One thing to do for in the future is to remove all whitespace between the macro name and argument. If spaces are needed, they're easily inserted like this: `$(MACRO $(EMPTY) ... any amount of whitespace ... and stuff`